spec/peep_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'peep'
+
+describe Peep do
+
+  describe '#all' do
```

Comment:
Minor note, but `#all` indicates an _instance method_, but this is a _class method_, which would be `::all`